### DISCUSSION TITLE
How to change node/edge/text color dynamically

### DISCUSSION THREAD

**Discussion by sathya-1994** [eclipse-glsp/glsp#762] [Q&A] [answered]
I am using GLSP's vscode extension.

I want to change node, edge and text color dynamically (edit the colors) and save these details in the .wf file.

Currently css classes related to colors are present in the webview.js file which we cannot edit dynamically.

**Comment by planger** [comment, marked as answer]
You could do that by creating custom view implementations that read certain attributes from the input GModel node (e.g. an attribute specifying the color code) and directly output it into the SVG elements' style attributes.
